gooddata-java/src/test/java/com/gooddata/sdk/service/project/ProjectServiceAT.java

```diff
@@ -144,6 +146,16 @@ public void addUsersToProject() {
         assertThat(user2, is(notNullValue()));
     }
 
+    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "addUsersToProject")
+    public void listProjectsForUser() {
+        final ProjectService projectService = gd.getProjectService();
+
+        final PageBrowser<Project> projects = projectService.listProjects(account1);
+
+        assertThat(projects, is(notNullValue()));
+        assertThat(projects.getPageItems(), contains(project));
+    }
+
     @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "addUsersToProject")
```

Comment:
```suggestion
    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "addUsersToProject")
    public void listProjectsForUser() {
        final ProjectService projectService = gd.getProjectService();

        final PageBrowser<Project> projects = projectService.listProjects(account1);

        assertThat(projects, is(notNullValue()));
        assertThat(projects.getPageItems().get(0).getTitle(), is(title));
    }

    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "addUsersToProject")
    public void listProjectsForUserSettingStartPage() {
        final ProjectService projectService = gd.getProjectService();

        final PageBrowser<Project> projects = projectService.listProjects(account1, new CustomPageRequest(10, 1));

        assertThat(projects, is(notNullValue()));
        assertThat(projects.getPageItems().isEmpty(), is(true));
    }

    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = {"listProjectsForUser", "listProjectsForUserSettingStartPage"})
```

Reply:
@cipous please check these updated ATs ;)

Reply:
ok, I dont see much benefit with the test of paging. Done as suggested.